packages/core/core-flows/src/order/workflows/cancel-order-fulfillment.ts

```diff
@@ -203,6 +203,7 @@ function prepareInventoryUpdate({
     inventory_item_id: string
     location_id: string
     quantity: BigNumberInput
+    line_item_id: string
```

Comment:
suggestion: Could we add a test for this?

Reply:
Added!